components/StyledButton.js

```diff
@@ -138,6 +138,7 @@ StyledButton.defaultProps = {
   buttonSize: 'medium',
   buttonStyle: 'standard',
   loading: false,
+  whiteSpace: 'nowrap',
```

Comment:
I don't think we should add this one. The real issue was the fixed width, and after thinking a bit more about it we should be careful with `whiteSpace=nowrap` because it can create some responsiveness issues with long strings. In normal conditions wrapping only happens when there's not enough place, and it's still better to have a multi-line button than to overflow the parent's width.